File: fizzbuzz.js
var x = 1;

while (x <= 100)
{
    if (x % 3 == 0)
        process.stdout.write("Fizz");
    if (x % 5 == 0)
        process.stdout.write("Buzz");
    if (x % 3 && x % 5)
        process.stdout.write(x.toString());
    process.stdout.write("\n");
    x++;
}

/*
** process.stdout.write() was used instead of console.log()
** to avoid automatic newlines at the end
*/

File: fizzbuzz.go
package main

import "fmt"

func main() {
    for x := 1; x <= 100; x++ {
        if (x % 3 == 0) {
            fmt.Printf("Fizz");
        }
        if (x % 5 == 0) {
            fmt.Printf("Buzz");
        }
        if (x % 3 != 0 && x % 5 != 0) {
            fmt.Printf("%d", x);
        }
        fmt.Printf("\n")
    }
}
